qa/L0_batcher/test.sh

```diff
@@ -403,10 +401,12 @@ done
 # by comparing the "response send" timestamps.
 TEST_CASE=test_multi_batch_preserve_ordering
 
-if [[ $BACKENDS == *"custom"* ]]; then
+# FIXME sleep_for does not delay for expected time when passing 400ms 
+# when using I/O is in GPU memory. Skip for Windows test.
+if [ "$OS_WINDOWS" -eq "0" ] && [ "$TEST_CUDA_SHARED_MEMORY" -eq 0 ]; then
```

Comment:
Why Windows is skipped? Is that because we don't build identity for Windows?

And I noticed that a lot of ensemble related tests are now disabled for Windows, but we should have been able to test that on Windows, right? Perhaps instead of using TF models, you should use ONNX addsub as ORT backend is supported on Windows.

Reply:
Yes, we do not build identity backend for windows. We earlier used the custom backend to signify if we were to test for models. The custom backend was not listed at the backend to test for on windows and hence these tests were earlier disabled as well. In short, no tests are being additionally disabled that were not already being run on windows.

Reply:
Can you file a ticket to address this and remove the Windows conditionals here and in other tests? I don't think we intended to skip Windows for ensemble tests and there is really no reason to do skip that (i.e. all parts needed for ensemble should be able to build for Windows)

Reply:
DLIS-2321 - Windows ensemble testing
DLIS-2315 - Cuda shared memory delay time failure